sdk/ml/azure-ai-ml/tests/command_job/unittests/test_command_job_entity.py

```diff
@@ -134,6 +135,7 @@ def test_command_job_builder_serialization(self) -> None:
             instance_type="STANDARD_BLA",
             timeout=300,
             code="./",
+            queue_settings=QueueSettings(job_tier="standard")
```

Comment:
Can we please include `priorty="medium"` as part of the test?

Reply:
Done